src/utils.ts

```diff
@@ -300,3 +300,55 @@ export const compactNumber = new Intl.NumberFormat('en', {
   //@ts-ignore
   compactDisplay: 'short',
 }).format;
+
+export const getMonthsByCycle = (
+  billingCycle: 'monthly' | 'quarterly' | 'half-yearly' | 'yearly',
+) => {
+  switch (billingCycle) {
+    case 'monthly':
+      return 1;
+    case 'quarterly':
+      return 3;
+    case 'half-yearly':
+      return 6;
+    case 'yearly':
+      return 12;
+    default:
+      return 1;
+  }
+};
+
+interface IPlan {
+  type: string;
+  credits: number;
+  subscriberLimit: number;
+  emailsByMonth: number;
+}
+
+export const getPlanDescription = (plan: IPlan) => {
+  const planDescription = {
+    descriptionId: 'plan_calculator.' + plan.type.replace('-', '_') + '_amount_description',
```

Comment:
```suggestion
    descriptionId: 'plans.' + plan.type.replace('-', '_') + '_amount_description',
```
How about something more general to rehuse?